microk8s-resources/actions/enable.kubeflow.sh

```diff
@@ -106,7 +106,7 @@ def get_hostname():
 
 def main():
     password = os.environ.get("KUBEFLOW_AUTH_PASSWORD") or get_random_pass()
-    bundle = os.environ.get("KUBEFLOW_BUNDLE") or "cs:kubeflow-195"
+    bundle = os.environ.get("KUBEFLOW_BUNDLE") or "cs:kubeflow-206"
```

Comment:
Could we use the syntax we have for passing arguments in addons (`microk8s enable kubeflow:bundle=lite`)?

Arguments (eg "bundle=lite") are passed as the first argument on the executable script [1].

[1] https://github.com/ubuntu/microk8s/blob/master/microk8s-resources/wrappers/microk8s-enable.wrapper#L39

Reply:
Given that the rest of the arguments for kubeflow are env vars, can we merge as-is, and open up a new PR that deprecates the env var arguments and switches to the string args?

Reply:
Sounds good. Lets work this on a new PR.